flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -187,7 +187,7 @@
 
 	private static final Time timeout = Time.milliseconds(10000L);
 
-	private String metricQueryServiceAddress;
```

Comment:
Why do you remove this variable? Is there any objection to retain this variable, just move metricRegistry a field, and close it after method? I think most of line add to get metricQueryServiceAddress below can be reduced.